object3d.cpp

```diff
@@ -24,3 +24,67 @@ Object3D& Object3D::operator =(const Object3D& oth) noexcept
 
 Object3D& Object3D::operator =(Object3D&& oth) noexcept
 { if(&oth != this) _colour = std::move(oth._colour); return *this; }
+
+void Object3D::setColor(const colour_t& colour) noexcept
+{ _colour = colour; }
+
+colour_t Object3D::getColor() const noexcept
+{ return _colour; }
+
+void Object3D::setScale(const glm::vec3& scale) noexcept
+{ _scale = scale; }
+
+glm::vec3 Object3D::getScale() const noexcept
+{ return _scale; }
+
+void Object3D::setRotate(const Quaternion& rotate) noexcept
```

Comment:
Change to setRotate(vec3 Axis, float angle)

Reply:
problem fixed